SIL.Windows.Forms/ImageGallery/ArtOfReadingImageCollection.cs

```diff
@@ -91,7 +162,7 @@ public int LoadMultilingualIndex(string pathToIndexFile)
 					Debug.Assert(parts.Length > defaultColumn);
 					if (parts.Length <= defaultColumn)
 						continue;
-					var partialPath = String.Format("{0}:AOR_{1}.png", parts[3], parts[1]);
+					var partialPath = GetPartialPath(rootImagePath, parts[3], parts[1], ref filenamePrefix, pathPrefix);
```

Comment:
Maybe it is too late on Friday, but it seems that partialPath is assigned to but never used.